pages/home/auth-providers/atlassian.mdx

```diff
@@ -21,15 +21,15 @@ This auth provider is used by:
 
 ## Configuring Atlassian auth
 
-At the moment, you can only configure Atlassian auth with a [self-hosted Engine](/home/install/overview).
-
-### Create a Atlassian app
+### Create an Atlassian app
 
 - Create a Atlassian app in the [Atlassian developer console](https://developer.atlassian.com/console/myapps/)
 - In the Authorization tab, click the "Add" action button and set the Callback URL to: `https://cloud.arcade.dev/api/v1/oauth/callback`
 - In the Permissions tab, enable any permissions you need for your app
 - In the Settings tab, copy the Client ID and Secret to use below
 
+Next, add the Atlassian app to your Arcade Engine configuration. You can do this in the Arcade Dashboard, or by editing the `engine.yaml` file directly (for a self-hosted instance).
```

Comment:
Added this text between "Create a {provider} app" and "Configuring {provider} auth" on every page. It felt nice to have a checkpoint that explained what the next step was.